tool/tsh/common/tsh.go

```diff
@@ -841,6 +841,8 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	proxyDB.Flag("cluster", clusterHelp).Short('c').StringVar(&cf.SiteName)
 	proxyDB.Flag("labels", labelHelp).StringVar(&cf.Labels)
 	proxyDB.Flag("query", queryHelp).StringVar(&cf.PredicateExpression)
+	proxyDB.Flag("request-reason", "Reason for requesting access").StringVar(&cf.RequestReason)
+	proxyDB.Flag("disable-access-request", "Disable automatic resource access requests").BoolVar(&cf.disableAccessRequest)
```

Comment:
In another PR I suggested a different flow for dealing with automatic access requests.
You can see it here: https://github.com/gravitational/teleport/pull/39003#discussion_r1515930078

That PR is not merged yet, but because of another reasons.
However, I think my suggestion applies here.

What do you think about it?

Reply:
Sounds good :+1: